### PR TITLE
Enhancement motor set acceleration

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
* Improve description of `wb_motor_set_acceleration`.
* Add links to function when referred from another section

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

File: reference/motor.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ Likewise, a [LinearMotor](linearmotor.md) can power a [SliderJoint](hingejoint.m
 - The `acceleration` field defines the default acceleration of the P-controller.
 It is expressed in *meter per second squared* (m/s²) for linear motors and in *meter per radian squared* (rad/s²) for rotational motors.
 A value of -1 (infinite) means that the acceleration is not limited by the P-controller.
-The acceleration can be changed at run-time with the `wb_motor_set_acceleration` function.
+Given that the `acceleration` value only specifies the maximum `acceleration` that can be achieved by the motor, other negative values than -1 are not supported.
```
Comment (omichel) [inline reference/motor.md]: Maybe I am wrong, but I would have written "negative values other than -1 are not supported" instead.

Comment (stefaniapedrazzi) [reply reference/motor.md]: Ok I will change it.
To me both are fine but they are slightly different and put the accent on different part of the sentence.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**